.github/workflows/nightly.yaml

```diff
@@ -10,12 +10,15 @@ on:
 jobs:
   plugins_test:
     name: Nightly Plugin Tests
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
     timeout-minutes: 60
     strategy:
       fail-fast: false
       matrix:
+        os: [ubuntu-latest, macos-latest]
         linter-version: [Snapshots, Latest]
```

Comment:
I was considering adding some config to prevent some Nightly jobs from running when the trigger is `pull_request` and the linter-version is `Latest`, since there is an identical run happening from `pr.yaml`, but not sure if that is too complex and/or straying from "repeatable runs" with those of nightly